pkg/controller/build/buildrequest/buildrequest_test.go

```diff
@@ -40,21 +40,28 @@ func TestBuildRequest(t *testing.T) {
 		unexpectedContainerfileContents []string
 	}{
 		{
-			name:     "With extensions image",
-			optsFunc: getBuildRequestOpts,
+			name: "With extensions image and extensions",
+			optsFunc: func() BuildRequestOpts {
+				opts := getBuildRequestOpts()
+				opts.MachineConfig.Spec.Extensions = []string{"usbguard"}
```

Comment:
usbguard is one of the more trivial extensions that contains only one RPM with the same name. Could we use e.g. the `ipsec` extension (which resolves to `NetworkManager-libreswan` and `libreswan` RPMs) for the test case instead?